Jenkinsfile

```diff
@@ -105,7 +105,7 @@ pipeline {
         stage('Unit Tests') {
```

Comment:
We need to clarify that unit tests are not expected for PR builds or else it's confusing to see it skipped in the pipeline, something like this:
```suggestion
        stage('Unit Tests for Release Builds') {
```